Update storage-import-export-service-faq.md

GabiSocoteanu opened the pull request:
confirmed with PM to update FAQ. we have many delays because of missing carrier info. customers need to know about potential delays and also help inform their carriers to collaborate.

GabiSocoteanu committed:
Update storage-import-export-service-faq.md

alkohli commented on articles/storage/common/storage-import-export-service-faq.md:
```diff
@@ -82,9 +82,18 @@ You can ship drives to the Azure datacenter using any known carrier like FedEx,
 - A FedEx account number in the US and EU, or
 - A DHL account number in the Asia and Australia regions.
 
+Note: India Datacenters require a Declaration letter on customer’s letterhead for returning the drives (Delivery Challan). You must also book the collection of the disks with your selected carrier and share the pickup details with the Datacenter in order to arrange the required entry pass. 
```
Change this to: >[!NOTE]
> The datacenters in India require a declaration letter on your letterhead (delivery challan) to return the drives. To arrange the required entry pass, you must also book the disks with your selected carrier and share the pickup details with the datacenter.

alkohli commented on articles/storage/common/storage-import-export-service-faq.md:
```diff
@@ -82,9 +82,18 @@ You can ship drives to the Azure datacenter using any known carrier like FedEx,
 ### Are there any restrictions with shipping my drive internationally?
 Please note that the physical media that you are shipping may need to cross international borders. You are responsible for ensuring that your physical media and data are imported and/or exported in accordance with the applicable laws. Before shipping the physical media, check with your advisors to verify that your media and data can legally be shipped to the identified data center. This will help ...
 
+### Are there any special requirements for delivering my disks to a DC?
```
DC is not an approved acronym for datacenter. Change to: ### Are there any special requirements for delivering my disks to a datacenter?

alkohli commented on articles/storage/common/storage-import-export-service-faq.md:
```diff
@@ -82,9 +82,18 @@ You can ship drives to the Azure datacenter using any known carrier like FedEx,
 
+### Are there any special requirements for delivering my disks to a DC?
+
+It depends on some specific Azure Datacenter restrictions.
```
Change the text to:
The requirements depend on the specific Azure datacenter restrictions. 

- There are a few Class C sites (for example, Frankfurt), that require a Microsoft datacenter Inbound ID number to be written on the parcel for security reasons. Before you ship your drives/disks to the datacenter, reach out to Azure Data Box Operations - *adbops@microsoft.com* - to get this number. Without this number, the package will be rejected.  
- The datacenters in India require the personal details of the driver such as the Government ID Card or Proof No. (for example, PAN, AADHAR, DL), name, contact, and the car plate number to get a gate entry pass. To avoid delivery delays, inform your carrier about these requirements.

alkohli reviewed (commented):
@GabiSocoteanu I have reviewed the proposed changes and suggested some changes myself. Please take a look and make those changes before we merge this request.

GabiSocoteanu committed:
Update storage-import-export-service-faq.md

made the changed proposed. in edit file view the alignment looks funny but I made it to look good it preview changes tab. 
in the qq: "Are there any special requirements for delivering my disks to a datacenter" the phrase was split in the middle and was creating a new row. 
I made it look good in preview. hope it remains ok. 
thank you

GitHubber17 committed:
edit text and format
